kubejs/server_scripts/enigmatica/kubejs/base/tags/items/forge/storage_blocks.js

```diff
@@ -59,4 +59,5 @@ onEvent('item.tags', (event) => {
     event.add(storageBlocks + '/glowstone', ['#chisel:glowstone']);
     event.add(storageBlocks + '/redstone', ['#chisel:redstone']);
     event.add(storageBlocks + '/coal', ['#chisel:coal']);
+	event.add(storageBlocks + '/glowstone', ['#chipped:glowstone']);
```

Comment:
I recommend adding the Prettier extension to Visual Studio Code (if you're using that) to autoformat on save, so avoid indentation errors like this :)

Reply:
I'm using Notepad++, I'll try to watch out and not spew janky identation all over next time though - or switch to something else if this becomes too much of a regular thing!